text.txt

```diff
@@ -0,0 +1 @@
+posted in Kusto Web Explorer / General at February 9, 2023 6:52:54 PM PST In case you missed it 👆 What's the email of copy editor?
```

Comment:
care to explain ?

Reply:
Woops, didn't mean to commit this. I'll delete it.